csharp/extractor/Semmle.Extraction.CSharp.DependencyFetching/DotnetCommand.cs

```diff
@@ -0,0 +1,61 @@
+using System.Collections.Generic;
+using System.Diagnostics;
+using Semmle.Util;
+
+namespace Semmle.Extraction.CSharp.DependencyFetching
+{
+    /// <summary>
+    /// Low level utilities to run the "dotnet" command.
+    /// </summary>
+    internal sealed class DotnetCommand : IDotnetCommand
```

Comment:
A bit of bike shedding. I find it confusing to have classes named `DotnetCommand` and `DotNet` too. Maybe `DotnetCliWrapper` or `DotnetCliInvoker` explains a bit better what this class represents.

Reply:
Totally fair points. I will do some renaming 👍

Reply:
Sorry, coming back on this topic. I actually thought about renaming only `DotnetCommand` to either `DotnetCliWrapper` or `DotnetCliInvoker`. You seem to have opted for also renaming `DotNet` to `DotNetCliWrapper`. If we go ahead with this change, should we also keep the consistency between the class and interface names (`DotNetCliWrapper` vs `IDotNet`)? Alternatively we could keep the original class name `DotNet`. What do you think?

Reply:
Haha, then I will rename `DotNetCliWrapper` to `DotNet` again :-) However, it actually made sense to name it `DotNetCliWrapper` because it wraps the invoker - also it makes sense to not rename the interface as it should not know about the concrete implementation (also made an effort to completely avoid referring to the concrete implementation types).

Reply:
Thanks. And sorry for the confusion.